contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -200,28 +206,54 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
 
         uint256 totalOperators = getNodeOperatorsCount();
         Packed64x4.Packed memory signingKeysStats;
-        for (uint256 operatorId; operatorId < totalOperators; ++operatorId) {
-            signingKeysStats = _loadOperatorSigningKeysStats(operatorId);
-            uint64 vettedSigningKeysCountBefore = signingKeysStats.get(VETTED_KEYS_COUNT_OFFSET);
-            uint64 totalSigningKeysCount = signingKeysStats.get(TOTAL_KEYS_COUNT_OFFSET);
-            uint64 depositedSigningKeysCount = signingKeysStats.get(DEPOSITED_KEYS_COUNT_OFFSET);
+        Packed64x4.Packed memory operatorTargetStats;
+        Packed64x4.Packed memory totalSigningKeysStats = _loadTotalSigningKeysStats();
+        uint64 vettedSigningKeysCountBefore;
+        uint64 totalSigningKeysCount;
+        uint64 depositedSigningKeysCount;
+        for (uint256 nodeOperatorId; nodeOperatorId < totalOperators; ++nodeOperatorId) {
+            signingKeysStats = _loadOperatorSigningKeysStats(nodeOperatorId);
+            vettedSigningKeysCountBefore = signingKeysStats.get(VETTED_KEYS_COUNT_OFFSET);
+            totalSigningKeysCount = signingKeysStats.get(TOTAL_KEYS_COUNT_OFFSET);
+            depositedSigningKeysCount = signingKeysStats.get(DEPOSITED_KEYS_COUNT_OFFSET);
 
             uint64 vettedSigningKeysCountAfter;
-            if (!_nodeOperators[operatorId].active) {
+            if (!_nodeOperators[nodeOperatorId].active) {
                 // trim vetted signing keys count when node operator is not active
                 vettedSigningKeysCountAfter = depositedSigningKeysCount;
             } else {
-                vettedSigningKeysCountAfter =
-                    Math64.min(totalSigningKeysCount, Math64.max(depositedSigningKeysCount, vettedSigningKeysCountBefore));
+                vettedSigningKeysCountAfter = uint64(Math256.min(totalSigningKeysCount, Math256.max(uint256(depositedSigningKeysCount), uint256(vettedSigningKeysCountBefore))));
             }
 
             if (vettedSigningKeysCountBefore != vettedSigningKeysCountAfter) {
                 signingKeysStats.set(VETTED_KEYS_COUNT_OFFSET, vettedSigningKeysCountAfter);
-                _saveOperatorSigningKeysStats(operatorId, signingKeysStats);
-                emit VettedSigningKeysCountChanged(operatorId, vettedSigningKeysCountAfter);
+                _saveOperatorSigningKeysStats(nodeOperatorId, signingKeysStats);
+
+                operatorTargetStats = _loadOperatorTargetValidatorsStats(nodeOperatorId);
```

Comment:
Why update targets only for operators with changed `vettedSigningKeysCount`?

Reply:
yep, it should be fixed